AngularApp/src/app/shared/services/diagnostic-api.service.ts

```diff
@@ -14,7 +14,7 @@ import { AdalService } from 'adal-angular4';
 @Injectable()
 export class DiagnosticApiService {
 
-  public readonly localDiagnosticApi: string = "http://localhost:5000/";
+  public readonly localDiagnosticApi: string = "http://localhost:22533/";
```

Comment:
Please revert this. Update your angular backed project so that it runs on port 5000 instead.

Reply:
Should I update: Applensv3\AngularApp\proxy.conf.json

Reply:
Make sure you are using this option:

![image](https://user-images.githubusercontent.com/7739017/48586689-77ceda80-e8e5-11e8-9409-6135f2435000.png)

Not sure what the proxy conf is.